docs/UserGuide.md

```diff
@@ -136,19 +139,17 @@ Format: `add appointment n/NAME r/REMARKS d/DATE t/TIME​`
 Examples:
 * `add appointment n/Meet Alex r/At M Hotel d/17-2-2021 t/1500`
 
-### Editing
-
-#### Editing a property : `edit property`
+### 3.3 Editing
 
-Overwrites the information of the property according to the flags provided.
+#### 3.3.1 Editing a property : `edit property`
 
-Formats: `edit property INDEX [n/NAME] [t/PROPERTY_TYPE] [a/ADDRESS] [p/POSTAL_CODE] [d/DEADLINE] [r/REMARKS] [cn/CLIENT_NAME] [cc/CLIENT_CONTACT_NUMBER] [ce/CLIENT_EMAIL] [ca/CLIENT_ASKING_PRICE] [tags/TAGS_SEPARATED_BY_COMMAS]​`
+Overwrites the information of the property according to the parameters provided.
```

Comment:
I think this doesn't sound very user friendly? A user may not know what a parameter is

Reply:
Will update this